### PR TITLE
Render a simple todo list

### PR DESCRIPTION
**Pull request by Darya-s**

### CODE REVIEW

File: react/week1/todolist/src/App.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+   <>
+    <TodoList todos={todos}/>
+    
+  
+
+     
+  
+     
+  </>
```
Comment (islanderupnorth) [inline react/week1/todolist/src/App.js:31]: Other than too much spacing when it's a single element it doesn't need to be enclosed by the React Fragment <></>

File: react/week1/todolist/src/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Todo from './Todo'
+export default function TodoList({todos}){
+    return(
+       todos.map(todo=> {
+           return <Todo key={todo.description} todo ={todo}/>
+       })
```
Comment (islanderupnorth) [inline react/week1/todolist/src/TodoList.js:7]: In this case when you want to do JavaScript inside of the HTML you need to wrap the JavaScript in {curly brackets} and here enclose it in the Fragment <></>

File: react/week1/todolist/src/Todo.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export default function todo({todo}){
+    return(
+        <div>
+         
+         <ul>
+            
+       <li> {todo.description} </li>
+        
+      <li>  {todo.deadline}</li>
+      </ul>
```
Comment (islanderupnorth) [inline react/week1/todolist/src/Todo.js:14]: The Todo item here should only be wrapped in 1 <li>{todo.description} {todo.deadline}</li>

You could use the <ul></ul> in the parent to wrap all the items

**Review by islanderupnorth — COMMENTED** [review]
Overall the structure is good and passing the right props to the right components. With the little tinkers throughout it should work as expected. It looks like you have understood the React fundamentals. Good work!

File: react/week1/todolist/src/TodoList.js (hunk 2 above)
Comment (Darya-s) [reply react/week1/todolist/src/TodoList.js:7]: I had wrapped it in curly brackets,is not that right? todo ={todo} ? yes,understood about fragment.

**Pull request merged**